scaleway/resource_domain_record.go

```diff
@@ -299,9 +335,8 @@ func resourceScalewayDomainRecordRead(ctx context.Context, d *schema.ResourceDat
 			return diag.FromErr(err)
 		}
 
-		for _, r := range res.Records {
-			record = r
-			break
+		if len(res.Records) > 0 {
```

Comment:
Why did this part changed?

Reply:
We were doing un unnecessary loop on Records.